modules/labs-react/expandable/lib/ExpandableAvatar.tsx

```diff
@@ -0,0 +1,27 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import {jsx} from '@emotion/react';
+
+import React from 'react';
+
+import {createComponent, ExtractProps, styled, StyledType} from '@workday/canvas-kit-react/common';
+import {Avatar} from '@workday/canvas-kit-react/avatar';
+import {space} from '@workday/canvas-kit-react/tokens';
+
+export interface ExpandableAvatarProps
+  extends Omit<
+    ExtractProps<typeof Avatar, never>,
+    keyof React.ButtonHTMLAttributes<HTMLButtonElement>
+  > {}
+
+const StyledAvatar = styled(Avatar)<StyledType>({
+  marginRight: space.xxs,
+  flexShrink: 0,
+});
+
+export const ExpandableAvatar = createComponent('div')({
```

Comment:
```suggestion
export const ExpandableAvatar = createComponent('button')({
```

This component should be 'button' because Avatar component is button element